src/bin/importSqlFile.js

```diff
@@ -71,14 +72,12 @@ const unescape = (val) => {
   await util.promisify(db.init.bind(db))();
   log('done');
 
-  log('open output file...');
-  const lines = fs.readFileSync(sqlFile, 'utf8').split('\n');
+  log(`Opening ${sqlFile}...`);
+  const stream = fs.createReadStream(sqlFile, {encoding: 'utf8'});
 
-  const count = lines.length;
+  log(`Reading ${sqlFile}...`);
   let keyNo = 0;
-
-  process.stdout.write(`Start importing ${count} keys...\n`);
-  lines.forEach((l) => {
+  for await (const l of readline.createInterface({input: stream, crlfDelay: Infinity})) {
```

Comment:
It appears this loop is never entered.
Placing a

    console.log('enter')

or

    log('enter')

in the line below is never evaluated, in this case shown on the screen.

Reply:
If I make the following change I see each line in the log, so it does appear to be reading the file correctly:
```diff
diff --git a/src/bin/importSqlFile.js b/src/bin/importSqlFile.js
index 148503e8..4146463d 100644
--- a/src/bin/importSqlFile.js
+++ b/src/bin/importSqlFile.js
@@ -78,6 +78,7 @@ const unescape = (val) => {
   log(`Reading ${sqlFile}...`);
   let keyNo = 0;
   for await (const l of readline.createInterface({input: stream, crlfDelay: Infinity})) {
+    console.log(l);
     if (l.substr(0, 27) === 'REPLACE INTO store VALUES (') {
       const pos = l.indexOf("', '");
       const key = l.substr(28, pos - 28);
```